tests/python/test_numpy_io.py

```diff
@@ -132,13 +132,8 @@ def test_numpy_io_example():
     # Vector
     arr = np.ones(shape=(n, m, 3), dtype=np.int32)
     vec.from_numpy(arr)
-    arr = np.ones(shape=(n, m, 3, 1), dtype=np.int32)
-    vec.from_numpy(arr)
 
     arr = vec.to_numpy()
-    assert arr.shape == (n, m, 3, 1)
-
-    arr = vec.to_numpy(as_vector=True)
     assert arr.shape == (n, m, 3)
```

Comment:
Now that we are using `keep_dims`, here we should test against `np.ones(shape=(n, m, 1, 3))` and `np.ones(shape=(n, m, 1, 1))` here as well.